docs/maatregelen/0-org-04-politiek-bestuurlijke-verantwoordelijkeheid.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Visie en politiek-bestuurlijke verantwoordelijkheid
+toelichting: Zorg voor een organisatievisie en politiek-bestuurlijk bewustzijn.
+levenscyclus:
+- organisatieverantwoordelijkheden
+rollen:
+- projectleider
+# - beleid-en-advies
+---
+<!-- tags -->
+
+
+## Maatregel
+Voor een passende algoritmegovernance is politiek-bestuurlijk bewustzijn, betrokkenheid en verantwoordelijkheid essentieel. 
+De kernvraag voor publieke organisaties bij de inzet van algoritmen is altijd: Hoe wegen we (als publieke organisatie of samenleving) de voordelen en nadelen van de inzet van algoritmen? 
+Dit is per definitie een kwalitatieve en politieke vraag. 
+Dit gaat niet alleen over direct opbrengsten maar ook over lange termijn en indirecte effecten, de mate waarin de inzet van technologie bijdraagt aan de legitimiteit van publieke organisatie en hoe burgers met deze technologie worden bejegend. 
+ 
+Een politiek-bestuurlijk kader is vervolgens om te zetten tot een uitgeschreven visie en aanpak om dit te concretiseren. Zie voorbeelden.
+
+## Aandachts- en actiepunten
```

Comment:
Dit hoofdstuk hebben we bij geen enkele maatregel. Daarom weghalen. De actie moet in de maatregel zelf. Je kan het eventueel verder uitsplitsen in de toelichting. 

Misschien is dus het opstellen van een visie of strategie ook een maatregel?

Reply:
Wat bedoel je met "Dit hoofdstuk hebben we bij geen enkele maatregel"? 

Eens dat het uit te splitsen is in politiek-bestuurlijk en visie, maar nu geen tijd voor.